nekoyume/Assets/_Scripts/Helper/Util.cs

```diff
@@ -26,6 +26,7 @@ public static class Util
     {
         public const int VisibleEnhancementEffectLevel = 10;
         private const string StoredSlotIndex = "AutoSelectedSlotIndex_";
+        private static readonly List<int> CrystalEquipmentRecipes = new() { 158, 159, 160 };
```

Comment:
Event recipes can be added in the future, so I recommend you to handle recipe informations in `ScriptableObject`